### PR TITLE
Gh 2189 fix imports master

### PR DESCRIPTION
**Pull request by hmottestad**
GitHub issue resolved: #2189  

Briefly describe the changes proposed in this PR:

 - remove unused imports
 - remove * imports
 - add curly braces

This will hopefully fix future merge conflicts with develop.

---- 
PR Author Checklist: 

 - [ ] my pull request is [self-contained](https://rdf4j.org/documentation/developer/merge-strategy/#self-contained-changes-pull-requests-and-commits)
 - [ ] I've added tests for the changes I made
 - [ ] every commit message starts with the issue number (GH-xxxx) followed by a meaningful description of the change
 - [ ] every commit has been [signed off](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for)

Note: we merge all feature pull requests using [squash and merge](https://help.github.com/en/github/administering-a-repository/about-merge-methods-on-github#squashing-your-merge-commits). See [RDF4J git merge strategy](https://rdf4j.org/documentation/developer/merge-strategy/) for more details.

### CODE REVIEW

File: compliance/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/LuceneSailTest.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 	/*
 	 * (non-Javadoc)
-	 *
```
Comment (hmottestad) [inline compliance/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/LuceneSailTest.java:20]: I'm sorry about all of these. Intellij feels the there shouldn't be a trailing space, and our style guide has no rule for this. I think Eclipse uses trailing spaces for javadocs.

**Comment by hmottestad** [comment]
I used Intellij's code and imports cleanup and then ran our formatter.

File: compliance/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/LuceneSailTest.java (hunk 1 above)
Comment (jeenbroekstra) [reply compliance/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/LuceneSailTest.java:20]: Fwiw I agree with IntelliJ here. I'll take a look at my eclipse settings to see if I can get it to remove these trailing whitespaces.

**Comment by jeenbroekstra** [comment]
@hmottestad can you make sure that when you merge, the title is correctly formatted? In particular the issue number. Here it's "Gh 2189" rather than "GH-2189", and github does not recognize that and doesn't link it.

**Comment by hmottestad** [comment]
We should have a CI action for that. I’ll look into it.

**Pull request merged**